checkout.py

```diff
@@ -4,13 +4,20 @@
 
 @author: vjham
 """
+from validate_clerk_input import validate_clerk_input
```

Comment:
potentially dumb question: Do we need this whole line? Or could we just do

`import validate_clerk_input`

Reply:
You might be right, I will test!

Reply:
I don't know the best practices or conventions in python. I thought maybe this was a best practice that you knew about.